samples/Common.c

```diff
@@ -21,6 +22,12 @@ VOID onDataChannelMessage(UINT64 customData, PRtcDataChannel pDataChannel, BOOL
     } else {
         DLOGI("DataChannel String Message: %.*s\n", pMessageLen, pMessage);
     }
+    // Send a response to the message sent by the viewer
```

Comment:
Is the customer expected to uncomment this to run? I am wondering if we can encapsulate all this into the `ENABLE_DATA_CHANNEL` directive which exists in the repository. That way we can avoid have commented code blocks.

Reply:
Added `ENABLE_DATA_CHANNEL` in necessary places